nilearn/plotting/img_plotting.py

```diff
@@ -595,6 +643,168 @@ def plot_roi(roi_img, bg_img=MNI152TEMPLATE, cut_coords=None,
     return display
 
 
+def plot_prob_atlas(maps_img, anat_img=MNI152TEMPLATE, view_type='auto',
+                    threshold=None, linewidths=2.5, cut_coords=None,
+                    output_file=None, display_mode='ortho',
+                    figure=None, axes=None, title=None, annotate=True,
+                    draw_cross=True, black_bg='auto', dim=False,
+                    cmap=plt.cm.gist_rainbow, vmin=None, vmax=None,
+                    alpha=0.5, **kwargs):
+
+    """ Plot the multiple atlas maps or statistical maps onto the anatomical image
+        by default MNI template
+
+        Parameters
+        ----------
+        maps_img: a nifti like object or the filename of the 4D probabilistic
+            atlas maps or statistical maps
```

Comment:
Break down this entry in the type, which goes on the first line, and what it means (4D probabilistic...) that goes on a line below.